### ISSUE TITLE
chore: add library manager workshop to main repo

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/CurriculumExpansion#620]
We need to take the following prototype here and break it down into steps and tests and at it to the main repo.

https://github.com/freeCodeCamp/CurriculumExpansion/pull/506/files

**Comment by Ksound22** [comment]
You're correct 💯

**Comment by Ksound22** [comment]
You're correct 💯

**Comment by jdwilkin4** [comment]
@Ksound22 

Don't forget to link this issue with your open PR so it will be closed automatically when merged in

**Comment by Ksound22** [comment]
@jdwilkin4 The number is referencing a different PR from way back. I think it's because the repos are different.

**Comment by jdwilkin4** [comment]
I updated your PR to link this issue to it.

When you are linking an issue from another repo, you need to copy the entire url like this

```md
Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/620
```

If you are linking to an issue within the same repo, then you can do `closes #620` 👍
